### DISCUSSION TITLE
第5节课作业(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#318] [作业]
16班第5节课作业提交在下面

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/tcit4ywas6pyfc7t?singleDoc#

**Comment by zszbyzsz** [comment]
https://juejin.cn/spost/7323867035636711465

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9257044

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/Dalld4SBpoUqidx3TtJcSbI9nBh?from=from_copylink

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch05

**Comment by Durian-1111** [comment]
**Day05-Homework**
链接：https://xby48gs45u.feishu.cn/docx/MautdIPWkoNcLCx3ap3c3aZNnfh?from=from_copylink

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/O8tek

**Comment by hhhanqian** [comment]
http://xhslink.com/YL6uLz

**Comment by Star-cre** [comment]
https://zhilengnuan.blog.csdn.net/article/details/135386192?spm=1001.2014.3001.5502

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17975465

**Comment by Bob-Y** [comment]
### 基础作业：

- 使用 LMDeploy 以本地对话、网页Gradio、API服务中的一种方式部署 InternLM-Chat-7B 模型，生成 300 字的小故事（需截图）

<img width="960" alt="a5bd2538e5fe5ef813e843d144aa145" src="https://github.com/InternLM/tutorial/assets/31715275/b0b83e28-3f13-47be-9eba-ecadb5186fc3">

<img width="960" alt="c70235875f2031646df774b6c6035a2" src="https://github.com/InternLM/tutorial/assets/31715275/1ab0dc93-948d-4a25-9597-ee381403a59d">

<img width="960" alt="a3d9567eba7d975a7cdf695934c563c" src="https://github.com/InternLM/tutorial/assets/31715275/c9ff339a-3376-4600-934d-38f29e1b18a7">

<img width="960" alt="d2a678dcca65941b20192b6b0bb079f" src="https://github.com/InternLM/tutorial/assets/31715275/514805d0-47af-463a-b0d5-bc9645c0be57">

<img width="960" alt="f56fe72f1d9a488be9d46349ac3cd17" src="https://github.com/InternLM/tutorial/assets/31715275/2c151eb6-5031-485d-ac6c-26a2e0fcfb79">

<img width="960" alt="ccbfb3343fd9f276f261761ed1a4bf1" src="https://github.com/InternLM/tutorial/assets/31715275/cc097404-d411-412e-b16a-7e7d1a7d2131">

- 量化

<img width="960" alt="08059a9b2352f1f2f170afec14895a1" src="https://github.com/InternLM/tutorial/assets/31715275/207bda86-c5b1-4cf4-9fcc-7dcf920ca95d">

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-05/
